packages/webapp-libs/webapp-finances/src/components/stripe/stripePaymentMethodSelector/__tests__/stripePaymentMethodSelector.component.spec.tsx

```diff
@@ -22,9 +28,11 @@ const StripePaymentMethodSelectorWithControls = () => {
 
 const Component = () => {
   return (
-    <Elements stripe={null}>
-      <StripePaymentMethodSelectorWithControls />
-    </Elements>
+    <CurrentTenantProvider>
```

Comment:
You don't need this provider here

Reply:
I think we need it. I could add it to the render util but since it's `webapp-finances` it does not have access to all the tenant providers. Not sure if I am supposed to add it directly to the rendering util or if it's better to add it to each test.

Reply:
Ok, got it now:/ I think we should modify the rendering utils in `webapp-finances` to it would use providers from `webapp-tenants`. I will take a look on this today